src/logger.js

```diff
@@ -2,8 +2,12 @@ const pino = require('pino');
 
 // Pretty debug logging
 let logger = pino({
-  prettyPrint: {},
-  prettifier: require('pino-colada'),
+  transport: {
+    target: 'pino-pretty',
```

Comment:
We should only do this `transport` option if we're in `debug` log level.

Reply:
Do we have env in satellite to run in `debug`  log level?

Reply:
Satellite honours whatever env vars are available.  We don't set them there, we set them in the Docker containers that run Satellite.  So you can use `process.env.LOG_LEVEL` and not worry about how it gets set.